README.md

```diff
@@ -1,21 +1,31 @@
+
 # DataBreaker
+[![JitPack](https://jitpack.io/v/SuperCoder7979/databreaker.svg)](https://jitpack.io/#SuperCoder7979/databreaker "SuperCoder7979/databreaker on JitPack")
 
 DataBreaker speeds up your minecraft loading time by stopping the execution of DataFixerUpper.
 As you can imagine this is very hacky and stops world backwards compatibility. This tool is intended for MOD DEVS ONLY.
 Do not use it in a normal Minecraft environment. Even if you try it will just crash you anyway.
 
-## how to add this mod to your dev env
+---
+
+## How to add this mod to your development environment:
 If you have the inclination to add databreaker to your dev env, follow these 3 easy steps:  
-1. add this to your build script before `dependencies`
-```
+
+### 1. add this to your build script before `dependencies`
+```gradle
 repositories {
-	maven { url = "https://jitpack.io" }
+	maven {
+		name = "JitPack"
+		url = "https://jitpack.io"
+	}
 }
 ```
-2. add this to your build script in `dependencies`:
-```
-modRuntime ("com.github.SuperCoder7979:databreaker:<whatever the latest commit hash is>") {
-	exclude module : "fabric-loader"
+
+### 2. add this to your build script in `dependencies`:
+###### If you don't want the latest build, you can find a list of builds on [JitPack](https://jitpack.io/#SuperCoder7979/databreaker "SuperCoder7979/databreaker on JitPack"). You can also use any git tagged version, or the (short or long) hash of any specific commit.
+```gradle
+modRuntime ("com.github.SuperCoder7979:databreaker:-SNAPSHOT") {
```

Comment:
`-SNAPSHOT` is generally not a good idea for regular dependencies (no build reproducibility, risk of suddenly getting a broken build, etc.). It would be better to recommend using the latest tag, as displayed by the jitpack widget.